python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -163,28 +163,53 @@ def location(self):
             tmpLocations = self.locations
         return tmpLocations
 
-    def get(self):
+    def get(self, localPath=''):
         """
         Retrieves locally all files matching this MassStorageFile object pattern
         """
+        self._actual_copy()
 
-        to_location = self.localDir
-
-        if not os.path.isdir(self.localDir):
-            if self._getParent() is not None:
-                to_location = self.getJobObject().outputdir
-            else:
-                logger.error("%s is not a valid directory.... Please set the localDir attribute" % self.localDir)
-                return
+    def _actual_copy(self, localPath=''):
```

Comment:
As I said in the PR I don't care for the naming scheme. I trust it's obvious what is going on here and I will happily take suggestions for better method names. Suggestions welcome.